PysalAnalysis.py

```diff
@@ -0,0 +1,14 @@
+from point_pattern import PointPattern
+import pysal as ps
+import random
+"""
+pysal "examples" gives an error
+ 
```

Comment:
I see your forum post, adding a grade for now and will update when you update this PR.